pkg/rancher-desktop/pages/Containers.vue

```diff
@@ -331,25 +331,27 @@ export default {
       return { content: sha };
     },
     getUniquePorts(ports) {
-      const keys = Object.keys(ports);
+      const displayMap = [];
 
-      const uniquePortMap = keys.map((key) => {
-        const values = ports[key];
-        const hostPorts = values.map(value => value.HostPort);
-        const uniqueHostPorts = [...new Set(hostPorts)];
+      Object.entries(ports).forEach((entry) => {
```

Comment:
Trivial style thing that doesn't block merging:
We don't really need a `forEach` here, right?
```suggestion
      for (const entry of Object.entries(ports)) {
```
Or even, given the next line:
```suggestion
      for (const [key, values] of Object.entries(ports)) {
```